powerauth-push-client/src/main/java/io/getlime/push/client/PushServerClient.java

```diff
@@ -139,7 +139,7 @@ public boolean createDevice(Long appId, String token, MobilePlatform platform, S
         request.setToken(token);
         request.setPlatform(platform.value());
         request.setActivationId(activationId);
-        TypeReference<Response> typeReference = new TypeReference<Response>() {
+        TypeReference<ObjectResponse> typeReference = new TypeReference<ObjectResponse>() {
```

Comment:
Why?

Reply:
Reason was because method called to map object from JSON and can't assign other type than ObjectResponse, see our method 
![image](https://user-images.githubusercontent.com/30933614/31909039-1df37ef4-b839-11e7-8183-f04929c163f6.png)

One solution is to make them all ObjectResponse.

Reply:
It is an API change - the change results in different objects being transmitted over the network. We cannot do API changes like this.

Reply:
Please look at difference between `Response` and `ObjectResponse`. With `Response`, there is no mapping...